src/tour.gleam

```diff
@@ -566,8 +569,7 @@ fn lesson_html(page: Lesson) -> String {
                 #("type", "button"),
                 #("alt", "Switch to light mode"),
                 #("title", "Switch to light mode"),
-                #("onclick", "setLightTheme()"),
-                #("class", "theme-button -light"),
+                #("class", "theme-button -light js-theme-button-light"),
```

Comment:
This is used for javascript rather than styling so let's use a data attribute here

Reply:
done